api/plugin_helpers.go

```diff
@@ -14,7 +14,7 @@ import (
 	"os"
 	"regexp"
 
-	squarejwt "gopkg.in/square/go-jose.v2/jwt"
+	squarejwt "github.com/go-jose/go-jose/v3/jwt"
```

Comment:
Just a suggestion: It could be nice to remove the import alias, since it is 1) unnecessary, and 2) no longer matches the naming of the library.

Reply:
Makes sense! I'm not entirely sure why it was there in the first place.